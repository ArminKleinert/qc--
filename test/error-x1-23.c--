
foo() {
    bits32 x;

    if (x) {
        ;
    }
        
    return;
}
