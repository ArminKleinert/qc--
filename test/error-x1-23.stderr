Character 28-52 Error: if-expression must be of type bool
